Tests fail with TypeError; BufferUtils is undefined

## Repro steps

64-bit Linux. `git clone`. `npm install`. `npm test`.

## Results

`builderror.log`:

    ../src/bufferutil.cc: In static member function ‘static void BufferUtil::Initialize(v8::Handle<v8::Object>)’:
    ../src/bufferutil.cc:32:17: warning: ‘v8::Local<v8::String> NanSymbol(const char*, int)’ is deprecated (declared at ../node_modules/nan/nan.h:1048) [-Wdeprecated-declarations]
         target->Set(NanSymbol("BufferUtil"), t->GetFunction());
                     ^
    ../src/bufferutil.cc:32:39: warning: ‘v8::Local<v8::String> NanSymbol(const char*, int)’ is deprecated (declared at ../node_modules/nan/nan.h:1048) [-Wdeprecated-declarations]
         target->Set(NanSymbol("BufferUtil"), t->GetFunction());
                                           ^
    ../src/validation.cc: In static member function ‘static void Validation::Initialize(v8::Handle<v8::Object>)’:
    ../src/validation.cc:112:17: warning: ‘v8::Local<v8::String> NanSymbol(const char*, int)’ is deprecated (declared at ../node_modules/nan/nan.h:1048) [-Wdeprecated-declarations]
         target->Set(NanSymbol("Validation"), t->GetFunction());
                     ^
    ../src/validation.cc:112:39: warning: ‘v8::Local<v8::String> NanSymbol(const char*, int)’ is deprecated (declared at ../node_modules/nan/nan.h:1048) [-Wdeprecated-declarations]
         target->Set(NanSymbol("Validation"), t->GetFunction());
                                       ^

On `npm test`, 172 pass, 72 fail.

First failure:

    1) Receiver can parse unmasked text message:
       TypeError: Cannot call method 'merge' of undefined
        at Receiver.concatBuffers (/home/an/builds/ws/lib/Receiver.js:305:14)
        at /home/an/builds/ws/lib/Receiver.js:467:38
        at Receiver.applyExtensions (/home/an/builds/ws/lib/Receiver.js:360:5)
        at /home/an/builds/ws/lib/Receiver.js:462:14
        at Receiver.flush (/home/an/builds/ws/lib/Receiver.js:336:3)
        at Receiver.opcodes.1.finish (/home/an/builds/ws/lib/Receiver.js:478:12)
        at Receiver.expectHandler (/home/an/builds/ws/lib/Receiver.js:453:31)
        at Receiver.add (/home/an/builds/ws/lib/Receiver.js:91:24)
        at Context.<anonymous> (/home/an/builds/ws/test/Receiver.test.js:18:7)
        at callFn (/home/an/builds/ws/node_modules/mocha/lib/runnable.js:250:21)
        at Test.Runnable.run (/home/an/builds/ws/node_modules/mocha/lib/runnable.js:243:7)
        at Runner.runTest (/home/an/builds/ws/node_modules/mocha/lib/runner.js:373:10)
        at /home/an/builds/ws/node_modules/mocha/lib/runner.js:451:12
        at next (/home/an/builds/ws/node_modules/mocha/lib/runner.js:298:14)
        at /home/an/builds/ws/node_modules/mocha/lib/runner.js:308:7
        at next (/home/an/builds/ws/node_modules/mocha/lib/runner.js:246:23)
        at Object._onImmediate (/home/an/builds/ws/node_modules/mocha/lib/runner.js:275:5)
        at processImmediate [as _immediateCallback] (timers.js:354:15)

Most other failures have similar results: they relate to to `BufferUtils.js` exporting `undefined`.

- - -

What's happening? (I don't know much about C++/node interop.) Ask for further diagnostics; will happily post.

--- a/src/bufferutil.cc
+++ b/src/bufferutil.cc
@@ -107,7 +107,7 @@ protected:
   }
 };
 
-extern "C" void init (Handle<Object> target)
+void init (Handle<Object> target)
 {
   NanScope();
   BufferUtil::Initialize(target);

--- a/src/validation.cc
+++ b/src/validation.cc
@@ -135,7 +135,7 @@ protected:
   }
 };
 
-extern "C" void init (Handle<Object> target)
+void init (Handle<Object> target)
 {
   NanScope();
   Validation::Initialize(target);

